### DISCUSSION TITLE
Rolling Pre-Release Questions

### DISCUSSION THREAD

**Discussion by PW5190** [Polprzewodnikowy/N64FlashcartMenu#151] [Q&A]
I have several questions about using the `rolling pre-release` build. Here's what I'm asking:
1. How do I add descriptions when viewing a ROM before launching it? The description tag says `Description: None.`
2. How do I change sound effects? Do I, perhaps, put the custom sounds on my SD Card?

I'm new to this build, so I'm still learning how to fully utilize it to its max potential.

**Comment by networkfusion** [comment]
You cannot do either. The prerelease is where we test new features before we release them. Not all features are fully implemented. You won't receive any support for the pre-release.
